keps/sig-ui/0035-add-plugin-support-to-dashboard.md

```diff
@@ -0,0 +1,154 @@
+---
+kep-number: 35
+title: Add plugin support to dashboard
+authors:
+  - "@ajatprabha"
+owning-sig: sig-ui
+participating-sigs:
+  - kubernetes-wide
+reviewers:
+  - "sig-ui-leads"
+  - "@maciaszczykm"
+approvers:
+  - "sig-ui-leads"
+  - "@maciaszczykm"
```

Comment:
We should just add @maciaszczykm to the `OWNER_ALIASES` instead https://github.com/kubernetes/enhancements/blob/master/OWNERS_ALIASES#L110